codegen/cmd_test.go

```diff
@@ -2062,7 +2062,7 @@ roleRef:
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: painter-{{ .Release.Name }}-{{ .Release.Namespace }}
+  name: painter-{{ .Release.Name }}-{{ .Release.Namespace }}-namespaced
```

Comment:
Can a `ClusterRole` be namespaced? Should we only add the `-namespaced` suffix to `Role` and `RoleBinding`?

Reply:
Namespaced refers to the usage of these resources, rather than the distinction between cluster wide and namespace specific

Reply:
That makes sense!